.github/workflows/github-ci.yml

```diff
@@ -45,17 +45,16 @@ jobs:
     strategy:
       matrix:
         node-version: [10.x]
-        project:
-          [
+        project: [
             frontend,
             api,
-            blockchain,
+            # blockchain,
```

Comment:
Consider removing if not used

Reply:
currently the blockchain project has some moderate vulnerabilities so the pipeline wouldn't go through.